### DISCUSSION TITLE
Send MessageLoaded every time Option

### DISCUSSION THREAD

**Discussion by igerzog** [postalserver/postal#1464] [Feature suggestions]
I think it will be useful to manually choose how to track opens - `on first open only` or `on every open`.

**Comment by iammattmartin** [comment]
Could be flawed though if you opt for "first" because some providers pre-load images via a proxy and that counts as "opened" before the message has even been seen.

**Comment by igerzog** [comment]
To tell the truth I need the second variant. Cause the open webhook is posted only one time now.
